dev/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -22,7 +22,7 @@ directive:
 
 ### Release
 
-``` yaml $(tag) == 'release_2022-07-01-preview'
+``` yaml $(tag) == 'release_2022-10-01-preview'
```

Comment:
You can't change this section. You need to add a new one for 10-01. Copy the 07-01 and change what you need.

Reply:
10-01 is the same as 07-01, I copied the section. May I learn how does this section of dev folder readme.md being used?